certificate-analyser/README.md

```diff
@@ -0,0 +1,46 @@
+##Certificate validation tool
+
+### Usage
+
+java -jar certificate-analyser-<version>.jar --help
```

Comment:
This is just me being hyper-picky : ) I'll leave it to your discretion.
https://wikidiff.com/analyser/analyzer

Reply:
I didnt know the difference, I think I will leave the bri`ish version here